pkgs/top-level/all-packages.nix

```diff
@@ -10594,7 +10594,7 @@ with pkgs;
 
   sssd = callPackage ../os-specific/linux/sssd {
     inherit (perlPackages) Po4a;
-    inherit (python27Packages) ldap;
+    inherit (python3Packages) ldap python;
```

Comment:
Why not use python3 and python3.pkgs.ldap?

Reply:
This way you can modify the python version with one change instead of two.
For example, pinning this to python 3.9 might look like
```diff
- inherit (python3Packages) ldap python;
+ inherit (python39Packages) ldap python;
```
instead of
```diff
- inherit (python3Packages) ldap;
- inherit python3
+ inherit (python39Packages) ldap;
+ inherit python39
```